dataland-frontend/src/components/pages/NewMissionPage.vue

```diff
@@ -0,0 +1,75 @@
+<template>
```

Comment:
Pls move to the other branch with stuff that will be implemented in the next ticket.